src/java.desktop/share/classes/javax/swing/ToolTipManager.java

```diff
@@ -270,8 +270,10 @@ void showTipWindow() {
 
             GraphicsConfiguration gc = getDrawingGC(toFind);
             if (gc == null) {
-                toFind = mouseEvent.getLocationOnScreen();
-                gc = getDrawingGC(toFind);
+                if (mouseEvent != null) {
```

Comment:
In the same method **showTipWindow**, mouseEvent's _getLocationOnScreen_ method is accessed at [L268](https://github.com/openjdk/jdk/blame/da3001daf79bf943d6194d9fd60250d519b9680d/src/java.desktop/share/classes/javax/swing/ToolTipManager.java#L268C13-L268C13) and mouseEvent's _getX_ and _getY_ methods are accessed at [L304](https://github.com/openjdk/jdk/blame/da3001daf79bf943d6194d9fd60250d519b9680d/src/java.desktop/share/classes/javax/swing/ToolTipManager.java#L304), guess we should guard them as well with null check.

Reply:
I had seen it but not sure in which condition it will fail as it had not failed until now in those codepaths..But Anyways, I agree and added the null check..